### PR TITLE
feat(oco): support new OCO orders

### PR DESCRIPTION
**Pull request by tony-ho**
Add support for "New OCO" order endpoint.

See: https://github.com/binance-exchange/binance-official-api-docs/blob/master/rest-api.md#new-oco-trade

### CODE REVIEW

File: src/http-client.js [hunk 1]
```diff
@@ -271,10 +283,11 @@ export default opts => {
     publicRequest: (method, url, payload) => pubCall(url, payload, method),
 
     order: payload => order(privCall, payload, '/api/v3/order'),
+    orderOco: payload => orderOco(privCall, payload, '/api/v3/order/oco'),
     orderTest: payload => order(privCall, payload, '/api/v3/order/test'),
     getOrder: payload => privCall('/api/v3/order', payload),
     cancelOrder: payload => privCall('/api/v3/order', payload, 'DELETE'),
-
+
```
Comment (balthazar) [inline src/http-client.js]: Could you remove this?

Comment (tony-ho) [reply src/http-client.js]: Oops, my diff tool must have been ignoring whitespace. 🤭Removed.

**Comment by tony-ho** [comment]
I have added an entry for the new endpoint to `README.md`.

I have a unit test I ran locally, however, it will place an order on the exchange as the Binance API doesn't currently provide a test endpoint for OCO orders.

Given this, there are a few options:
- Place an order with an invalid parameter (eg. quantity) or that won't meet the Binance price filters. This should still provide code coverage as the endpoint will be exercised, but the order will fail and won't be executed.
- Place a valid order, then cancel it. This will require the user to have funds to cover the order, and that the order parameters are valid and conservative enough that the order won't be filled before it is cancelled.

Let me know if you would like either of those options implemented, or if you have an alternative approach.

**Comment by balthazar** [comment]
This looks good! Thank you!

**Comment by balthazar** [comment]
The absence of test is fine then, I don't want to do 2 as it could potentially be executed, and 1 isn't so helpful. I guess maybe since you had to create a new function maybe it could? But I won't block the PR on this.

**Comment by jsappme** [comment]
any plan to support OCO margin orde anytime soon?

**Pull request merged**